modules/installation-guide/partials/proc_enabling-dev-workspace-operator.adoc

```diff
@@ -32,18 +32,25 @@ spec:
 +
 [subs="+quotes,+attributes"]
 ----
-$ {prod-cli} server:deploy --che-operator-cr-patch-yaml=patch.yaml ...
+$ {prod-cli} server:deploy --workspace-engine=dev-workspace ...
 ----
-+
-`patch.yaml` must contain the following:
-+
+
+ifeval::["{project-context}" == "che"]
+[WARNING]
+====
+{prod-cli} will automatically setup Dex as OIDC provider on Minikube. For other {kubernetes} clusters setup link:https://kubernetes.io/docs/reference/access-authn-authz/authentication/#configuring-the-api-server[{kubernetes} OIDC] provider following cluster provider documentation and set the following values in {prod-checluster} Custom Resource (CR):
```

Comment:
setup Dex as the OIDC provider on Minikune

Reply:
fixed